src/main/java/com/SAS/game_event_logger/GameEventLogger.java

```diff
@@ -0,0 +1,27 @@
+package com.SAS.game_event_logger;
+
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.Comparator;
+import java.util.List;
+
+public class GameEventLogger {
+    private String gameID;
+    private LocalDate gameDate;
+    private List<GameEvent> eventList;
+
+    public GameEventLogger(String gameID, LocalDate gameDate) {
+        this.gameID = gameID;
+        this.gameDate = gameDate;
+        this.eventList = new ArrayList<>();
+    }
+
+    public void addNewEvent(GameEvent event) {
+        eventList.add(event);
+    }
+
```

Comment:
I think you need to add removeEvent as well, so if there was a mistake or something it can be removed.

Reply:
I think so as well, will add it for feautrue Task